packages/shared/components/popups/LedgerTransaction.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+    import { Illustration, Text } from 'shared/components'
+    import { displayNotifications, showAppNotification } from 'shared/lib/notifications';
+    import { formatUnitBestMatch } from 'shared/lib/units';
+    import { get } from "svelte/store";
+
+    export let locale
+
+    export let onCancel = () => {}
+
+    export let remainderAddress = ''
+    export let remainderAmount = null
+    let shouldDisplayRemainderAddress = remainderAddress.length > 0
+    let shouldDisplayRemainderAmount = remainderAmount !== null
+
+    export let toAddress = ''
+    export let toAmount = null
+    let shouldDisplaySendTo = toAddress.length > 0 && toAmount !== null
+
+    let isInvalid = false
+
+    const onInvalid = () => {
+        /**
+         * NOTE: Because getLocaleData is used multiple times, we need to make sure
+         * that the user isn't spammed with error messages for when the transaction data
+         * is invalid.
+         */
+        if(!isInvalid) {
+            isInvalid = true
```

Comment:
How come we are setting is to `true` here? Do we set it to `false` somewhere else?

Reply:
I'm adding more info to the comment above it, but it is basically a check to ensure that the notification only gets shown once. It is not set to false anywhere else, which is intended in this case.